src/test/java/org/junit/AssumptionViolatedExceptionTest.java

```diff
@@ -92,4 +115,93 @@ public void canSetCauseWithInstanceCreatedWithExplicitThrowableConstructor() {
       AssumptionViolatedException e = new AssumptionViolatedException("invalid number", cause);
       assertThat(e.getCause(), is(cause));
     }
+
+    @Test
+    public void assumptionViolatedExceptionWithoutValueAndMatcherCanBeReserialized_v4_13()
+            throws IOException, ClassNotFoundException {
+        assertExceptionReserializable(new AssumptionViolatedException(MESSAGE));
+    }
+
+    @Test
+    public void assumptionViolatedExceptionWithValueAndMatcherCanBeReserialized_v4_13()
+            throws IOException, ClassNotFoundException {
+        assertExceptionReserializable(new AssumptionViolatedException(MESSAGE, TWO, SERIALIZABLE_IS_THREE));
+    }
+
+    @Test
+    public void unserializableValueAndMatcherCanBeSerialized() throws IOException, ClassNotFoundException {
+        AssumptionViolatedException exception = new AssumptionViolatedException(MESSAGE,
+                UNSERIALIZABLE_VALUE, UNSERIALIZABLE_MATCHER);
+
+        assertAssumptionViolatedExceptionCanBeSerialized(exception);
+    }
+
+    @Test
+    public void nullValueAndMatcherCanBeSerialized() throws IOException, ClassNotFoundException {
+        AssumptionViolatedException exception = new AssumptionViolatedException(MESSAGE);
+
+        assertAssumptionViolatedExceptionCanBeSerialized(exception);
+    }
+
+    @Test
+    public void serializableValueAndMatcherCanBeSerialized() throws IOException, ClassNotFoundException {
+        AssumptionViolatedException exception = new AssumptionViolatedException(MESSAGE,
+                TWO, SERIALIZABLE_IS_THREE);
+
+        assertAssumptionViolatedExceptionCanBeSerialized(exception);
+    }
+
+    private void assertAssumptionViolatedExceptionCanBeSerialized(AssumptionViolatedException exception)
```

Comment:
A shorter name is easier to read and it is consistent with JUnit's own assertions (e.g. assertTrue instead of assertBooleanTrue). This is a suggestion and not mandatory for approval.

```suggestion
    private void assertCanBeSerialized(AssumptionViolatedException exception)
```